roles/deploy/tasks/main.yml

```
@@ -80,6 +80,8 @@
     job: "logrotate {{ tuxedo_log_rotation_config_path }}"
     cron_file: /etc/crontab
 
+- include_tasks: nfs.yml
+
```

It's likely we will want this to be optional dependent on the presence of configuration for NFS mounts:

```suggestion
- include_tasks: nfs.yml
  when: nsfmounts is defined and nsfmounts | length > 0
```